dockerfiles/platform-build/Dockerfile

```diff
@@ -71,7 +71,7 @@ RUN mkdir build
 WORKDIR /hpcc-dev/build
 
 ARG SIGN_MODULES=OFF
-RUN --mount=type=secret,id=key,uid=10000 if [[ "${SIGN_MODULES}" != "OFF" ]] ; \
```

Comment:
This defaults to signing on (e.g. if SIGN_MODULES is not set).  Is that sensible?  Should it be checking SIGNING_SECRET?